### PR TITLE
tailscale: Add version 1.34.2

### PR DESCRIPTION
**Pull request by shayne**
Tailscale a WireGuard based mesh VPN, used
to connect your computers and other devices
together securely without proxies.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by shayne** [comment]
@tech189 any chance I could get 👀 on this?

File: bucket/tailscale.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        }
+    },
+    "post_install": [
+        "sudo tailscale-ipn.exe /install"
+    ],
+    "uninstaller": {
+        "script": [
+            "Stop-Process -Name 'tailscale-ipn' -Force -ErrorAction SilentlyContinue | Out-Null",
+            "sudo tailscale-ipn.exe /uninstall"
+        ]
```
Comment (tech189) [inline bucket/tailscale.json:28]:
```suggestion
        "script": [
            "if (!(is_admin)) { error 'Admin rights are required to uninstall'; break }"
            "Stop-Process -Name 'tailscale-ipn' -Force -ErrorAction SilentlyContinue | Out-Null",
            "tailscale-ipn.exe /uninstall"
        ]
```

See my notes above.

File: bucket/tailscale.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ],
+    "checkver": {
+        "url": "https://pkgs.tailscale.com/stable/",
+        "regex": "tailscale-setup-([\\d.]+)-amd64\\.msi"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://pkgs.tailscale.com/stable/tailscale-setup-$version-amd64.msi"
```
Comment (tech189) [inline bucket/tailscale.json:49]: Are there any checksum hashes available for these downloads?

**Review by tech189 — COMMENTED** [review]
Thanks for your contribution! Please take a look at my review comments and re-run checkver to bring the manifest up-to-date.

File: bucket/tailscale.json (hunk 2 above)
Comment (shayne) [reply bucket/tailscale.json:51]: I added it to our pkgs server and will update the JSON once deployed.

Comment (shayne) [reply bucket/tailscale.json:51]: I pushed the commit including the hash URL. Still need to roll out pkgs server change.

Comment (tech189) [reply bucket/tailscale.json:51]: Thank you, this will help to speed up autoupdates! Let me know when it's live and then I can merge.

Comment (shayne) [reply bucket/tailscale.json:51]: @tech189 Ready to go! pkgs.tailscale.com now offers sha256, for example:
https://pkgs.tailscale.com/stable/tailscale-setup-1.34.2-amd64.msi.sha256

**Pull request merged**